### DISCUSSION TITLE
Firewall Settings

### DISCUSSION THREAD

**Discussion by DJ-3ncoder** [Security-Onion-Solutions/securityonion#11466] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

48

### RAM

256

### Storage for /

2.18 TB

### Storage for /nsm

1.88 TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I set up the Security Onion and there are no errors or warnings in so-status at all but the airgap installation wants to connect to a certain IP all the time:

SecurityOnionIP -> 169.254.169.254:80 and :443

This data comes from the firewall log-file which I cannot post here.

My first question:
It seems that the firewall config is intended to be done via webinterface and not via shell because _systemctl status firewalld_ gives back "dead". Did I get this correctly? What firewall is used when firewalld is inactive?

My second question:
How can i disable these outgoing connections? What even are these connections?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment, marked as answer]
For firewall rules iptables is used

169.254.169.254 is an internal IP used in cloud service providers for hosting metadata. This is not a routable IP address
